src/coreclr/nativeaot/Runtime/i386/AsmMacros.inc

```diff
@@ -163,6 +163,10 @@ G_EPHEMERAL_HIGH                            equ _g_ephemeral_high
 G_CARD_TABLE                                equ _g_card_table
 RhpWaitForGC2                               equ @RhpWaitForGC2@4
 RhpTrapThreads                              equ _RhpTrapThreads
+RhpStressGc                                 equ @RhpStressGc@0
+RhpGcPoll2                                  equ @RhpGcPoll2@4
+RhHandleGet                                 equ @RhHandleGet@4
+RhpGcSafeZeroMemory                         equ @RhpGcSafeZeroMemory@8
```

Comment:
`RhpGcSafeZeroMemory` does not appear to be used anywhere in the asm helpers.

Reply:
(Also, `RhHandleGet`)

Reply:
Nice catch... they are not... it was supposed to come up in the next PR about calling conventions but I accidentally left it here while doing the split. I'll remove it for now.

For the curious readers - there's an issue with name mangling and calling conventions. I'll have a long write up about possible solutions, their pros and cons. I prototyped two different solutions and one of them depends on the [alternatename](https://devblogs.microsoft.com/oldnewthing/20200731-00/?p=104024) linker/pragma trick. The downside of the trick is that linker doesn't see the comment pragma if there's no other reference to the .obj file inside a static library. There happened to be exactly two files that have only `COOP_PINVOKE_HELPER` and no other referenced code. As a workaround I added a dummy reference from the .asm files for one function from each of those two files.